### DISCUSSION TITLE
Payment Popup Form Branding

### DISCUSSION THREAD

**Discussion by FlashInThePan** [Glench/ExtPay#109] [General]
Maybe I've missed something but I don't seem to see a way to upload my extension/company logo to the payment/trial popups. Is this possible?

**Comment by burkybang** [comment]
Just write code in your extension to stylize that page however you want 🙂

**Reply by FlashInThePan** [reply]
Thanks for the response. Do you have any examples?

**Reply by burkybang** [reply]
No, I don't have an example. You would use a content script and/or a style sheet referenced in your manifest.
